velox/docs/functions/spark/math.rst

```diff
@@ -0,0 +1,57 @@
+====================================
+Mathematical Functions
+====================================
+
+.. spark:function:: abs(x) -> [same as x]
+
+    Returns the absolute value of ``x``.
+
+.. spark:function:: add(x, y) -> [same as x]
+
+    Returns the result of adding x to y. The types of x and y must be the same.
+    For integral types, overflow results in an error. Corresponds to sparks's operator ``+``.
+
+.. spark:function:: ceil(x) -> [same as x]
+
+    Returns ``x`` rounded up to the nearest integer.  
+    In spark only long, double, and decimal have ceil.
+
+.. spark:function:: divide(x, y) -> [same as x]
+
+    Returns the results of dividing x by y. The types of x and y must be the same.
+    It always performs floating point division. Corresponds to spark's operator ``/``.
```

Comment:
> It always performs floating point division.

Do you mean that dividing by zero doesn't raise an error even if input types are integers? Let's clarify.

Reply:
I mean that the result will always be double.
```
SELECT 3 / 2; -- 1.5
SELECT 2L / 2L; -- 1.0
```

Reply:
Then, [same as x] is incorrect. It needs to be double. It would still be nice to clarify the behavior when y is zero.

`.. spark:function:: divide(x, y) -> [same as x]`

->

`.. spark:function:: divide(x, y) -> double`

Reply:
Just added example.
SELECT 3 / 0; -- NULL